### PR TITLE
added a freqoperations file that will contain freq methods

### PR DESCRIPTION
**Pull request by abuebola**
for now there's only PSD

### CODE REVIEW

File: core/freqtools/freqoperations.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
+from scipy.signal import welch
+
+
+# Compute the PSD of a time series
+# Input: samplesarray, sampling frequency
+# Returns: array of frequency bins, PSD
+def compute_psd(samplesarray, fs):
+    fbins, pxx = welch(samplesarray, fs=fs, nperseg=int(min((fs, len(samplesarray)))))
+    return fbins, pxx
+
```
Comment (niroigen) [inline core/freqtools/freqoperations.py]: You can remove line 10, 11, 12

File: core/freqtools/freqoperations.py [hunk 2]
```diff
@@ -0,0 +1,12 @@
+from scipy.signal import welch
+
+
+# Compute the PSD of a time series
+# Input: samplesarray, sampling frequency
+# Returns: array of frequency bins, PSD
+def compute_psd(samplesarray, fs):
+    fbins, pxx = welch(samplesarray, fs=fs, nperseg=int(min((fs, len(samplesarray)))))
```
Comment (niroigen) [inline core/freqtools/freqoperations.py]: What are`fbins` and `pxx` suppose to mean?

File: core/freqtools/freqoperations.py [hunk 3]
```diff
@@ -0,0 +1,12 @@
+from scipy.signal import welch
+
+
```
Comment (niroigen) [inline core/freqtools/freqoperations.py]: There should just be one line between code and imports

Comment (abuebola) [reply core/freqtools/freqoperations.py]: https://stackoverflow.com/questions/2953250/python-pep8-blank-lines-convention

File: core/freqtools/freqoperations.py (hunk 1 above)
Comment (abuebola) [reply core/freqtools/freqoperations.py]: pycharm complains if I do that, saying it doesnt respect pep8 conventions, lol

**Pull request merged**